assets/microsoft.yaml

```diff
@@ -913,23 +913,76 @@
   monitoring: unknown
   feedback: https://huggingface.co/microsoft/Orca-2-13b/discussions
 - type: model
-  name: Phi-3 Mini
+  name: Phi-3
   organization: Microsoft
-  description: Phi-3 Mini is a 3.8 billion-parameter, lightweight, state-of-the-art open model trained using the Phi-3 datasets.
-  created_date: 2024-04-23
-  url: https://azure.microsoft.com/en-us/blog/introducing-phi-3-redefining-whats-possible-with-slms/
-  model_card: https://huggingface.co/microsoft/Phi-3-mini-128k-instruct
+  description: Phi-3 is a 14 billion-parameter, lightweight, state-of-the-art open
+    model trained using the Phi-3 datasets.
+  created_date: 2024-05-21
+  url: https://arxiv.org/abs/2404.14219
+  model_card: https://huggingface.co/microsoft/Phi-3-medium-128k-instruct
   modality: text; text
-  analysis: The model has been evaluated against benchmarks that test common sense, language understanding, mathematics, coding, long-term context, and logical reasoning. The Phi-3 Mini-128K-Instruct demonstrated robust and state-of-the-art performance among models with fewer than 13 billion parameters.
-  size: 3.8B parameters
+  analysis: The model has been evaluated against benchmarks that test common sense,
+    language understanding, mathematics, coding, long-term context, and logical
+    reasoning. The Phi-3 Medium-128K-Instruct demonstrated robust and state-of-the-art
+    performance.
+  size: 14B parameters
   dependencies: []
   training_emissions: unknown
-  training_time: 7 days
-  training_hardware: 512 H100-80G GPUs
-  quality_control: The model underwent post-training processes viz. supervised fine-tuning and direct preference optimization to increase its capability in following instructions and aligning to safety measures.
+  training_time: unknown
```

Comment:
We should make more obvious that the 3.8B param model discloses this info but the 14B doesnt